packages/plugins/VuetifyExtended/lib/components/VexMiniMeta/VexMiniMeta.vue

```diff
@@ -27,8 +28,8 @@
     },
     computed: {
       stackProps() {
-        return getBindableProps.call(this, VexStack.props)
+        return getBindableProps.call(this, VexStack.props);
```

Comment:
```suggestion
        return getBindableProps.call(this, VexStack.props);
```
пропущено options. проверить поиском как минимум все вхождения getBindableProps